yaml/xyz/openbmc_project/Inventory/Item/PCIeSlot.interface.yaml

```diff
@@ -98,6 +104,46 @@ enumerations:
           description: >
               Type of the PCIe slot is unknown
 
+    - name: Status
+      description: >
+          Possible link status
+      values:
+          - name: 'Operational'
+            description: >
+                Link is operational
+
+          - name: 'Degraded'
+            description: >
+                Link is degraded
+
+          - name: 'Unused'
+            description: >
+                Link is unused
+
+          - name: 'Deactivated'
+            description: >
+                Link is deactivated
+
+          - name: 'Failed'
+            description: >
+                Link failed
+
+          - name: 'Open'
+            description: >
+                Link is open
+                
+          - name: 'Inactive'
+            description: >
+                Link is inactive
+
+          - name: 'Unused'
+            description: >
+                Link is unused
+
```

Comment:
I don't see Unused and Unknown Link status being used in the python script. Probably there is a good reason for these two status, and must have discussed in the design meetings. So, how/when we will be using/setting these states?